lib/daru/view/templates/googlecharts/static_html.erb

```diff
@@ -0,0 +1,7 @@
+<html lang='en'>
+<head>
+  <title>Chart</title>
+  <%= initial_script %>
+</head>
+<body><%= chart_script %></body>
```

Comment:
I prefer 
```
<body>
   <%= chart_script %>
</body>

``` 
            
  It already comes in bunch of js code. So alignment is important.